drivers/telescope/rainbow.cpp

```diff
@@ -922,8 +947,17 @@ bool Rainbow::Abort()
 /////////////////////////////////////////////////////////////////////////////
 bool Rainbow::Sync(double ra, double dec)
 {
-    char cmd[DRIVER_LEN] = {0};
 
+    if (SaveAlignBeforeSyncS[0].s == ISS_ON)
+    {
+           StarAlign(ra, dec);
+    }
+    else
+    {
+        LOG_DEBUG("SaveAlignBeforeSyncS is OFF. Not doing anything.");
+    }
```

Comment:
so starAlign and regular sync can be run after each other? or ONLY one should be used at a time?

Reply:
yes you're definitely correct, Chris also mentioned this so I changed the command and give the status printout updated a little bit